apps/website/locales/ar.ts

```diff
@@ -0,0 +1,283 @@
+      translators: "المترجمين",
+      avatar: {
+        tooltip: false
+      }
+    },
+    downloads: {
+      title: "تحميل",
+      steps: {
+        install: false,
+        login: false,
+        add: false,
+        showoff: false
+      },
+      section: {
+        heading: {
+          title: "حان الوقت الإظهار.",
+          description: "استخدم PreMiD الآن وأظهر لأصدقائك ما تفعله ، ربما تجد شخصًا لديه نفس الاهتمامات.",
+          getStarted: "إبدء الان",
+          extension: "الملحق"
+        }
+      },
+      browser: {
+        your: false,
+        other: false,
+        based: false
+      },
+      mobile: {
+        title: false,
+        description: false
+      },
+      alphaAccess: {
+        title: false,
+        description: false,
+        callToAction: false
+      },
+      faq: false,
+      faqs: {
+        q1: {
+          question: false,
+          answer: false
+        },
+        q2: {
+          question: false,
+          answer: false
+        },
+        q3: {
+          question: false,
+          answer: false
+        },
+        q4: {
+          question: false,
+          answer: false
+        },
+        q5: {
+          question: false,
+          answer: false
+        },
+        q6: {
+          question: false,
+          answer: false
+        },
+        q7: {
+          question: false,
+          answer: false
+        },
+        q8: {
+          question: false,
+          answer: false
+        },
+        q9: {
+          question: false,
+          answer: false
+        }
+      }
+    },
+    store: {
+      title: "المتجر",
+      noPresence: false,
+      presence: {
+        button: {
+          reportIssue: false,
+          suggestFeature: false,
+          viewCode: false
+        },
+        title: {
+          description: "الوصف",
+          information: "معلومات"
+        },
+        informationSection: {
+          contributors: false,
+          version: false,
+          users: false,
+          tags: false,
+          supportedUrls: false
+        }
+      },
+      header: {
+        categories: "الفئات",
+        search: "البحث على presence"
+      }
+    }
+  },
+  footer: {
+    partners: "الشركاء",
+    followUs: false,
+    supportUs: "ادعمنا",
+    more: "المزيد",
+    legal: false,
+    supportList: {
+      donate: "التبرع",
+      contribute: "ساهم",
+      translate: "الترجمة"
+    },
+    moreList: {
+      faq: false,
+      documentation: "الوثائق",
+      status: "الحالة"
+    },
+    legalList: {
+      privacyPolicy: "سياسة الخصوصية",
+      termsOfService: false,
+      cookiePolicy: "سياسة ملفات تعريف الإرتباط"
+    },
+    withLoveBy: false,
+    by: false,
+    copyright: false
+  },
+  error: {
+    404: {
+      title: false,
+      message: false
+    },
+    500: {
+      title: false,
+      message: false
+    },
+    default: {
+      title: false,
+      message: false,
+      button: false
+    }
+  }
+}));
```

Comment:
Wouldn't it be better to do this? (2/2)
```suggestion
}));
export default defineI18nLocale()
```